source/chapters/user_interface.rst

```diff
@@ -644,9 +644,13 @@ PFL/Headphone Button
    The headphone buttons of both decks in the mixer
 
 The headphone button is also known as the :term:`pre-fader listen or PFL <PFL>`
-button. Pressing this button allows you to listen and synchronize the track you
-will play next in your headphones before your audience hears the track
-(headphone cueing). You can select more than one PFL button simultaneously.
+button. Pressing this button plays the track on the headphone output at full
```

Comment:
Can you add the sentence at the end. The main usage is the headphone-cueing which should be explained first IHMO. Useful note should come after that.